otaclient/app/boot_control/_cboot.py

```diff
@@ -485,6 +486,13 @@ def pre_update(self, version: str, *, standby_as_ref: bool, erase_standby=False)
 
     def post_update(self):
         try:
+            # firmware update
+            firmware = Firmware(
+                self.standby_slot_mount_point
+                / Path(cfg.FIRMWARE_CONFIG).relative_to("/")
+            )
+            firmware.update(self._cboot_control.get_standby_slot() == "0")
```

Comment:
This line is kind of ambiguous, what about directly passing the slot number(0 or 1) into the update function?